technical-reports/resolver/introduction.md

```diff
@@ -1,10 +1,3 @@
 # Introduction
 
-Design tokens are key-value pairs that represent design decisions, allowing for consistent use across different platforms and tools. In complex design systems, there is a need to manage multiple dimensions such as themes, modes (e.g., light and dark), brands, and variants. This introduces challenges in organizing tokens, resolving references, and applying overrides.
-
-The Resolver Specification addresses these challenges by introducing a structured mechanism to:
-
-- Organize tokens into **sets** and **modifiers**.
-- Define how tokens are **resolved** when multiple sets and modifiers are involved.
-- Support **scalability** by handling arbitrary dimensions without complicating the core token definitions.
-- Maintain **modularity** and **reusability** by allowing tokens to be combined and overridden in a controlled manner.
+This module describes the means by which design tokens may express multiple values for the purposes of “theming” (light mode, dark mode, high contrast mode, etc.) or other scenarios.
```

Comment:
Proposal: removing all opinions, undefined terms, and summary statements, this describes in plain language what this module does: define how multiple values for a single design token cna be expressed.

The only term introduced here is “theming,” but it is immediately followed by examples. If we want, we can define this further.

“Other scenarios” is a vague, undefined clause we may also define if we want; this could be improved.